IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -661,7 +669,7 @@ - (BOOL)saveToken:(MSIDBaseToken *)token
     
     if (!key)
     {
-        [self stopTelemetryEvent:event withItem:cacheItem success:NO context:context];
+        [self stopTelemetryEvent:event withItem:cacheItem success:NO logWipeData:NO context:context];
```

Comment:
nit: I see logWipeData:YES is only called in 2 places. Perhaps we can have a method that logs wipe data on top of stopTelemetryEvent or separated?

Reply:
Sure :)

Reply:
done